### ISSUE TITLE
[6.2.0-beta.14] Unknown exception created by Scroller

### ISSUE THREAD

**Issue by matheo** [storybookjs/storybook#14214]
**Describe the bug**

I was trying to see if the next version had fixed this issue:

![image](https://user-images.githubusercontent.com/260185/110873656-36c21780-82a0-11eb-91c4-60eb82192891.png)

but instead I got:

![image](https://user-images.githubusercontent.com/260185/110873381-a08df180-829f-11eb-838e-c5b80311a84b.png)

**To Reproduce**

I've updated my packages from `~6.1.21` to `~6.2.0-beta.14`
Do I need to update something in my webpack config or something?

**System**

```
System:
    OS: Linux 5.4 Ubuntu 20.04.2 LTS (Focal Fossa)
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
  Binaries:
    Node: 12.18.3 - ~/.nvm/versions/node/v12.18.3/bin/node
    Yarn: 1.22.5 - /usr/bin/yarn
    npm: 6.14.6 - ~/.nvm/versions/node/v12.18.3/bin/npm
  Browsers:
    Chrome: 89.0.4389.82
    Firefox: 86.0
  npmPackages:
    @storybook/addon-a11y: ~6.2.0-beta.14 => 6.2.0-beta.14 
    @storybook/addon-actions: ~6.2.0-beta.14 => 6.2.0-beta.14 
    @storybook/addon-docs: ~6.2.0-beta.14 => 6.2.0-beta.14 
    @storybook/addon-links: ~6.2.0-beta.14 => 6.2.0-beta.14 
    @storybook/addon-viewport: ~6.2.0-beta.14 => 6.2.0-beta.14 
    @storybook/addons: ~6.2.0-beta.14 => 6.2.0-beta.14 
    @storybook/angular: ~6.2.0-beta.14 => 6.2.0-beta.14
```

**Additional context**

I've removed my custom panel and my `6.1` installation seems to not work with `6.2` :thinking:

**Comment by shilman** [comment]
Did you try removing `node_modules` and/or lockfiles and reinstalling?

**Comment by matheo** [comment]
@shilman yep, I doubled checked and I did refresh the `node_modules` when updating it :)

**Comment by ayhid** [comment]
I have the same issue with 6.2.7 :/ @matheo  did you solve the problem ?

**Comment by matheo** [comment]
Nope, I'm still waiting a deep solution :(
cc @shilman

**Comment by shilman** [comment]
Do you have a repro repo you can share?

**Comment by carlosbaraza** [comment]
I'm getting this error too. It happens intermittently, sometimes when I change the `main.js` or `preview.js` it will start appearing. And sometimes it will just work, which makes it quite strange.

![Screenshot 2021-04-19 at 21 03 25](https://user-images.githubusercontent.com/1270425/115296417-d81b7380-a152-11eb-8feb-a58396d3cb66.png)

**Comment by carlosbaraza** [comment]
Not sure if this has fixed it. But I updated `react` to 17.0.2 (was on 17.0.1) and `react-dom`, and now it seems to be stable 🤞

**Comment by matheo** [comment]
Thanks for the tip @carlosbaraza !

**Comment by matheo** [comment]
Didn't work for my repo :disappointed: 
(I see that `@storbook/angular` relies on `react@16.14` but I guess it doesn't affect the manager, dunno)

![image](https://user-images.githubusercontent.com/260185/115315714-82889c00-a13d-11eb-8d55-605884df29a3.png)

**Comment by shilman** [comment]
@matheo that dep is for the manager. we can try updating it in 6.3 and see if that fixes your problem?

**Comment by matheo** [comment]
yes please @shilman :slightly_smiling_face:

**Comment by evelynhathaway** [comment]
Data point: We were already on react and react-dom v17, but upgrading to the v6.3.0 storybook beta and downgrading back to stable worked. Possibly it was an issue with our cached manager or node module resolutions.

**Comment by matheo** [comment]
@shilman I've upgraded to `@storybook/angular@6.3.0-beta.4` and it still uses `react-dom@16.14.0` therefore I have no way to confirm that the exception comes from that one, I still have it :(

**Comment by gavmck** [comment]
I had some conflicting versions of react which I'm not sure contributed to this. I ended up having to delete my `package-lock.json` and `node_modules` folder and re-run `npm install`.

The error initially surfaced during a mass upgrade of dependencies to try and bring things up to date.

**Comment by matheo** [comment]
I finally found a solution while upgrading to `Storybook v6.3.9`!
To fix my `package.json` to `React v16`:
```
    "react": "16.14.0",
    "react-is": "16.13.1",
 ```

**Comment by ChazUK** [comment]
I am also getting this error, I tried using React 17.0.2 and 16.14.0 but still getting the Scroller error.

Environment Info:

  System:
    OS: macOS 10.15.7
    CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
  Binaries:
    Node: 14.17.0 - /usr/local/bin/node
    Yarn: 2.4.3 - ~/.npm-global/bin/yarn
    npm: 6.14.13 - /usr/local/bin/npm
  Browsers:
    Chrome: 94.0.4606.113
    Safari: 15.1
  npmPackages:
    @storybook/addon-actions: ^6.1.15 => 6.3.12 
    @storybook/addon-backgrounds: ^6.1.15 => 6.3.12 
    @storybook/addon-controls: ^6.1.15 => 6.3.12 
    @storybook/addon-docs: ^6.1.15 => 6.3.12 
    @storybook/addon-jest: ^6.1.15 => 6.3.12 
    @storybook/addon-knobs: 6.1.15 => 6.1.15 
    @storybook/addon-viewport: ^6.1.15 => 6.3.12 
    @storybook/angular: ^6.1.15 => 6.3.12 
    @storybook/react: ^6.1.15 => 6.3.12

**Comment by ChazUK** [comment]
Sorry, nuked my node_modules folder and switch to 16.14.0 and it's now working!

**Comment by Pringels** [comment]
I have the same issue. (@storybook/preact: `^6.3.12`, Preact `10.5.14`) 

Tried nuking node_modules and lockfiles but nothing works.
